nautobot_ssot/tests/test_contrib.py

```diff
@@ -309,6 +309,68 @@ class Adapter(NautobotAdapter):
             "Custom fields aren't properly loaded through 'BaseAdapter'.",
         )
 
+    def test_default_get_queryset(self):
+        """Test default 'get_queryset' method."""
+
+        class TenantModel(NautobotModel):
+            """Test model for testing default 'get_queryset' method."""
+
+            _model = Tenant
+            _modelname = "tenant"
+            _identifiers = ("name",)
+            _attributes = ("description",)
+
+            name: str
+            description: str
+
+        class Adapter(NautobotAdapter):
+            """Test adapter for testing default 'get_queryset' method."""
+
+            top_level = ("tenant",)
+            tenant = TenantModel
+
+        new_tenant_name = "NASA"
+        Tenant.objects.create(name=new_tenant_name)
+        adapter = Adapter()
+        adapter.load()
+        diffsync_tenant_1 = adapter.get(TenantModel, new_tenant_name)
+        diffsync_tenant_2 = adapter.get(TenantModel, "Test")
+
+        self.assertEqual(new_tenant_name, diffsync_tenant_1.name)
+        self.assertEqual("Test", diffsync_tenant_2.name)
+
+    def test_overwrite_get_queryset(self):
+        """Test overriding 'get_queryset' method."""
+
+        class TenantModel(NautobotModel):
+            """Test model for testing overridden 'get_queryset' method."""
+
+            _model = Tenant
+            _modelname = "tenant"
+            _identifiers = ("name",)
+            _attributes = ("description",)
+
+            name: str
+            description: str
+
+            @classmethod
+            def get_queryset(cls):
+                return Tenant.objects.filter(name__startswith="N")
+
+        class Adapter(NautobotAdapter):
+            """Test overriding 'get_queryset' method."""
+
+            top_level = ("tenant",)
+            tenant = TenantModel
+
+        new_tenant_name = "NASA"
+        Tenant.objects.create(name=new_tenant_name)
+        adapter = Adapter()
+        adapter.load()
+        diffsync_tenant = adapter.get(TenantModel, new_tenant_name)
+
+        self.assertEqual(new_tenant_name, diffsync_tenant.name)
```

Comment:
Maybe we can create another tenant that falls outside of the filter and ensure it isn't loaded?